symplyphysics/laws/thermodynamics/inner_energy_from_sum_heat_and_work.py

```diff
@@ -10,12 +10,16 @@
 # Formula: dU = Q + A
 # Where:
 # dU is the change in internal energy of the system,
-# A is the work done by the system, 
+# A is the work done by the system,
 # Q is the heat supplied to the system.
```

Comment:
Descriptions for both laws are identical, but one is with minus sign and another with plus. Both cannot be true, so one of the descriptions is false.